src/Autofac.Pooling/PoolActivator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Autofac.Core;
+using Autofac.Core.Resolving.Pipeline;
+using Microsoft.Extensions.ObjectPool;
+
+namespace Autofac.Pooling
+{
+    /// <summary>
+    /// An activator for creating new <see cref="ObjectPool{TLimit}"/> instances.
+    /// </summary>
+    /// <typeparam name="TLimit">The limit type of the pool items.</typeparam>
```

Comment:
This comment seems to be different with every `TLimit` reference... 😆

Reply:
🙄